assets/dev/scss/frontend/widgets/image-box.scss

```diff
@@ -67,32 +43,38 @@
 		}
 	}
 
-	@media (max-width: $screen-mobile-max) {
-
-		// TEMP code to override the image spacing
-		.elementor-image-box-img {
-			margin-left: auto !important;
-			margin-right: auto !important;
-			margin-bottom: 15px;
-		}
-	}
-
 	.elementor-image-box-img {
 		display: inline-block; // For alignment content
 	}
 
+	&.elementor-element .elementor-widget-container .elementor-image-box-wrapper .elementor-image-box-img {
+		margin-left: var(--image-margin-left, var(--image-spacing, 15px));
```

Comment:
I have added the additional classnames names for CSS specificity reasons. 

I need to overwrite this general styling code: 
.elementor .elementor-widget:not(.elementor-widget-text-editor):not(.elementor-widget-theme-post-content) figure {
    margin: 0;
}